src/lib/components/wizard-navigation-bar.component.horizontal.less

```diff
@@ -126,41 +111,40 @@
       .state-circle(@width, @height, @border-width);
       .state-default(@wz-color-default);
     }
-    &.current .step-indicator {
-      .state-default(@wz-color-current);
+    &.optional .step-indicator {
+      .state-default(@wz-color-optional);
     }
-    &.done .step-indicator {
+    &.done .step-indicator,
+    &.completed .step-indicator {
       .state-default(@wz-color-done);
     }
-    &.optional .step-indicator {
-      .state-default(@wz-color-optional);
+    &.current .step-indicator {
+      .state-default(@wz-color-current);
     }
-    &.editing .step-indicator {
+    &.current.done .step-indicator,
+    &.current.completed .step-indicator {
       .state-default(@wz-color-editing);
     }
 
-    a:hover .step-indicator {
+    &.navigable a:hover .step-indicator {
       .state-circle-hover(@width, @height, @border-width);
       .state-hover(@wz-color-default);
     }
-    &.current a:hover .step-indicator {
-      .state-hover(@wz-color-current);
+    &.navigable.optional a:hover .step-indicator {
+      .state-hover(@wz-color-optional);
     }
-    &.done a:hover .step-indicator {
+    &.navigable.done a:hover .step-indicator,
+    &.navigable.completed a:hover .step-indicator {
       .state-hover(@wz-color-done);
```

Comment:
I think I would prefer a deeper nested form, because it feels easier to read I think:
```less
&.navigable.done, &.navigable.completed {
   a:hover .step-indicator {
      .state-hover(@wz-color-done);
   }
}
```

Reply:
Done in bc3b71f5